Assets/Scripts/Capabilities/Jump.cs

```diff
@@ -64,6 +69,12 @@ private void FixedUpdate()
         if (isGrounded)
         {
             jumpPhase = 0;
+            if (jumpState != JumpState.Grounded)
+            {
+                animator.SetBool(Constants.JUMP_FALL_ANIMATION, false);
+                animator.SetBool(Constants.JUMP_LAND_ANIMATION, true);
+            }
+            jumpState = JumpState.Grounded;
```

Comment:
Having two variables keep track of the grounded state (`isGrounded`, `jumpState.Grounded`) might introduce some bugs if we're not careful. Could we replace `isGrounded` with this new jumpState? No worries if that would be too much effort.

Reply:
I can probably do that